azure/azure_style.yaml

```diff
@@ -132,3 +135,31 @@ jobs:
 
           find . -iname '*.h' -o -iname '*.hpp' -o -iname '*.c' -o -iname '*.cpp' | xargs clang-format --verbose --dry-run --Werror -style=file
 
+  - job: fprettify
+    pool:
+      vmImage: "ubuntu-20.04"
+    timeoutInMinutes: ${{ parameters.TIMEOUT }}
+    condition: ${{ parameters.FPRETTIFY }}
+    steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: "3.9"
+      - script: |
+          # Install fprettify
+          pip install fprettify==0.3.7
+
+          # Run fprettify on non-differentiated Fortran 90 source files
+          find . -type f -iname '*.f90' ! -iname '*_b.f90' ! -iname '*_d.f90' -print0 | while read -d $'\0' file
+          do
+            echo Checking $file
+            fprettify -i=4 -l=120 --whitespace-multdiv=True --strict-indent $file
```

Comment:
Do we want to stick to command line args, or move to a config file since it's supported to fprettify? That may make it easier for end users (similar to flake8 etc.)

Reply:
I am fine with switching to a config file because we do have a few args here. I just tried it out locally and the config file seems to work.